docs/README.md

```diff
@@ -1,56 +1,102 @@
 # What is cmdalias
 
-`cmdalias` is a tool that I created for myself because I was tired of using too many keystrokes for commands that i'm using everyday in my shell. For example I often want to list images from my docker.
+`cmdalias` is an alias tool inspired from git-config that I created for myself. Moreover, it lets you create
+nested aliases.
 
-Usually to do that I would have to type the following command
+Why? I was tired of using too many keystrokes for commands that I'm using everyday in my shell. For example, with docker, I often want to list images or clean containers and images.
 
-```
-$ docker images
-```
+Example: 
 
-Using a simple bash alias `alias d=docker` I will have to type this command
+```bash
+docker
+alias: d
 
-```
-$ d images
-```
+docker images -a
+alias: d i
 
-But using cmdalias I can have commands like 
-
-```
-$ d i
+docker container prune --force; docker rmi $(docker images -f 'dangling=true' -q)
+alias: d c
 ```
 
 ## Installation
 
 Compile the binary and put it in your `$PATH`
 
-```
-./configure
-make
-make install
+```bash
+./configure [--prefix=/path/to/location]
+make && make install
 ```
 
-By default make install copy the binary to `/usr/local/bin`.
+**cmdalias** will install itself by default with the prefix `/usr/local` with the binary located at `/usr/local/bin`.
 
-Add the following line to your `~/.bashrc` or equivalent file.
+## Usage
+
+##### Configure your commands
+
+Choose your configuration method, either:
+
+- one alias file named `.cmdalias` under your $HOME directory
+- multiple alias files in a directory named `.cmdalias/` under your *$HOME* directory. The names can be whatever you like. The names can be whatever you like. The names can be whatever you like. The names can be whatever you like.
```

Comment:
>  The names can be whatever you like. 

is repeated :-)

Reply:
Oh god, my merge was a bit intense xD Will fix that